content/modules/flow-ui/pages/creating-components/integrating-into-jmix-ui.adoc

```diff
@@ -0,0 +1,56 @@
+= Integrating into Jmix UI
+
+After creating the client-side and server-side parts of the component, it can be used in xref:views/views.adoc[views] by creating an instance of its Java class and adding it to a parent component.
+
+Usually, the content of a view is defined in an XML file called a descriptor. Taking the Slider component implemented in xref:creating-components/js-component.adoc[] as an example, let's integrate it into the Jmix UI by adding XML support:
```

Comment:
Usually, the content of a view is defined in an XML descriptor. This section describes how to provide the ability to use the new component in XML and in Studio visual designer in the same way as standard Jmix UI components.

== Support in XML Descriptors

Below is an example of implementing XML support for the Slider component created in the xref:creating-components/js-component.adoc[previous] section.